content/en/docs/reference/access-authn-authz/abac.md

```diff
@@ -8,6 +8,7 @@ title: Using ABAC Authorization
 content_template: templates/concept
 weight: 80
 ---
+{{< note >}}This ABAC Authorization feature has been considered deprecated from Kubernetes 1.6 release{{< /note >}}
```

Comment:
Perhaps instead use the feature-state shortcode.  And I believe if you put in the {{% capture overview %}} section it will show up at the top of the page instead of the bottom.  So instead do the following. 
{{< note >}}
{{< feature-state state="deprecated" for_k8s_version="1.6" >}}
The ABAC Authorization feature has been considered deprecated from the Kubernetes 1.6 release.
{{< /note >}}